.github/workflows/docker.yaml

```diff
@@ -22,10 +22,6 @@ jobs:
           go-version: 1.14
         id: go
 
-      - name: Setup statik
-        run: |
-          go get github.com/rakyll/statik
-
```

Comment:
@nopcoder we don't need statik to build the Docker image?

Reply:
make gen should build and download the tool if needed